pkgs/tools/networking/tox-node/default.nix

```
@@ -1,42 +1,30 @@
-{ lib, rustPlatform, fetchFromGitHub
-, libsodium, openssl
-, pkg-config
-, fetchpatch
+{ lib, rustPlatform, fetchFromGitHub, nix-update-script
 }:
 
 with rustPlatform;
 
 buildRustPackage rec {
```

Not really related to this bump, but we really commonly write this as:

```suggestion
rustPlatform.buildRustPackage rec {
```